docs/Reference/CLI/CLI-Subcommands.md

```diff
@@ -174,4 +174,26 @@ This data is included in the [IBFT 2.0 genesis file](../../HowTo/Configure/Conse
         
         ``` tab="RLP Output"
         0xf853a00000000000000000000000000000000000000000000000000000000000000000ea94be068f726a13c8d46c44be6ce9d275600e1735a4945ff6f4b66a46a2b2310a6f3a93aaddc0d9a1c193808400000000c0
-        ```
\ No newline at end of file
+        ```
+
+## retesteth
+
+This command enables Retesteth-compatibilty, and accepts the following command line options:
```

Comment:
```suggestion
Runs a Retesteth compatible server.  This command accepts the following command line options:
```

I think the first _this command_ is redundant and splitting into two sentences simplifies the language.